scipy/stats/_levy_stable/c_src/levyst.c

```diff
@@ -0,0 +1,93 @@
+#include <stdlib.h>
+#define _USE_MATH_DEFINES
+#include <math.h>
+#include "levyst.h"
```

Comment:
This should include `<stdlib.h>` to properly declare `malloc`.  As it stands, this works incidentally because if `malloc` is not explicitly declared it is implicitly declared to return `int`, which you are then casting below. C doesn't require `malloc` to be cast because it's weak type system allows implicit conversion of void pointers. Casting `malloc` hides the lack of inclusion of `<stdlib.h>`, but this can actually lead to undefined behavior in some cases.

```suggestion
#define _USE_MATH_DEFINES
#include <math.h>
#include <stdlib.h>
#include "levyst.h"
```

Reply:
Good spot! Thanks